pkgs/os-specific/linux/systemd/0002-Don-t-try-to-unmount-nix-or-nix-store.patch

```diff
@@ -1,18 +1,20 @@
-From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Eelco Dolstra <eelco.dolstra@logicblox.com>
-Date: Fri, 12 Apr 2013 13:16:57 +0200
+From 400dfb2405f42da99c37a9a5c00edd2dd56e7cc1 Mon Sep 17 00:00:00 2001
```

Comment:
Hmm could we keep the original commit authors in these patches please?

Reply:
Nevermind. Original-Author is still there

Reply:
Let me know if this is not enough. It's a bit annoying to go ahead and change all the patches manually to the original author because I was using git to reflow everything.